apps/full-site-editing/dev-plugin/dev.php

```diff
@@ -1,16 +1,12 @@
 <?php
 /**
- * Plugin Name: Full Site Editing Development
- * Description: Overrides for Full Site Editing development.
+ * Plugin Name: WordPress.com Editor Toolkit Development
```

Comment:
Should this be "Editing" instead of "Editor"?

Reply:
fixed